### PR TITLE
Issue#98 - URL is being injected (strong dependency removed) and most…

### PR DESCRIPTION
**Pull request by samshub**
Ready for review...

### CODE REVIEW

File: pkg/db/mongodb.go [hunk 1]
```diff
@@ -19,25 +13,23 @@ var (
 
 // MongoDB encapsulates a mongo session with all relevant collections
```
Comment (gabelula) [inline pkg/db/mongodb.go]: That comment should remove "with all relevant collections"

File: pkg/db/mongodb.go [hunk 2]
```diff
@@ -19,25 +13,23 @@ var (
...
-	Authors        *mgo.Collection
-	Sections       *mgo.Collection
-	TagTargets     *mgo.Collection
-	CayUserActions *mgo.Collection
-	SearchHistory  *mgo.Collection
+	Session *mgo.Session
+	DB      *mgo.Database
 }
 
 //Close closes the mongodb session; must be called, else the session remains open
```
Comment (gabelula) [inline pkg/db/mongodb.go]: idem. Fix comment.

File: pkg/service/action.go [hunk 3]
```diff
@@ -18,18 +18,18 @@ func ImportAction(context *web.AppContext) (*model.Action, *web.AppError) {
 		return nil, err
 	}
 
-	if err := setReferences(context.DB, &input); err != nil {
+	if err := setReferences(context.MDB, &input); err != nil {
```
Comment (gabelula) [inline pkg/service/action.go]: why from .DB to .MDB ?

File: pkg/db/mongodb.go (hunk 1 above)
Comment (samshub) [inline pkg/db/mongodb.go]: Fixed. See below.

File: pkg/service/action.go (hunk 3 above)
Comment (samshub) [inline pkg/service/action.go]: MDB goes with struct `MongoDB`.

**Comment by gabelula** [comment]
Nice. Ready to go!

**Pull request merged**